src/datasources/users/entities/wallets.entity.db.ts

```diff
@@ -0,0 +1,58 @@
+import { RowSchema } from '@/datasources/db/v1/entities/row.entity';
+import {
+  Column,
+  Entity,
+  Unique,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  JoinColumn,
+  Index,
+} from 'typeorm';
+import { z } from 'zod';
+import { getAddress } from 'viem';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
+import { User } from '@/datasources/users/entities/users.entity.db';
+
+export const WalletSchema = RowSchema.extend({
+  address: AddressSchema,
+});
+
+@Entity('wallets')
+@Unique(['user', 'address'])
```

Comment:
This will make the combination of the two unique.
```suggestion
@Unique(['user'])
@Unique( ['address'])
```

Reply:
Would this mean that a user can't appear twice in different rows?
According to the [spec](https://www.notion.so/safe-global/Users-can-create-an-account-15e8180fe5738029ad5dc9141672bbd1?pvs=4#1628180fe5738002bf6ef8f793c6f6f4) a user can have multiple wallet entries so I think we need to make sure that the same user and wallet combination must be unique.

Reply:
Although the current constraint would allow different users to have the same address. Maybe it just needs to be `@Unique(['address'])` then so that address is unique but users can appear multiple times in the table?

Reply:
@usame-algan My comment was more about the usage of the `@unique` decorator. Sure, if the logic needs the address field to be the only unique column, we should remove `User` from the constraint.